src/service/resource.js

```diff
@@ -265,7 +265,7 @@ export class Resource {
       this.state_ = ResourceState.NOT_LAID_OUT;
     }
     if (opt_callback) {
```

Comment:
I'm wondering where the callbacks are coming from. We're calling `callback(false)` in `Resources`, so leaving it to `Resource` to call `callback(true)` is a bit odd. Can we remove `opt_callback` entirely?